feat(privacy): Add Storage Access Policy error pages to sidebar (#39789)
mdn/content: files/en-us/web/privacy/guides/firefox_tracking_protection/index.md

sidebar: privacy
---

> This page describes how Firefox's built-in tracking protection works and how loading of third-party content is affected.
> Developers should consider this information to ensure compatibility when this feature is enabled by visitors.

Reviewer: Nits and a rephrase suggestion, mostly to switch to active voice and to directly address devs:
```suggestion
This page describes how Firefox's built-in tracking protection works and how it affects loading of third-party content.
You should consider how this feature works to ensure your site works correctly when visitors have tracking protection enabled.
```
